.gitlab-ci.yml

```diff
@@ -9,7 +9,7 @@ variables:
   GIT_SUBMODULE_STRATEGY: recursive
   SINGULARITY_EOS_GCC_VERSION: "9.3.0"
   SINGULARITY_EOS_OPENMPI_VERSION: "4.1.3"
-  SINGULARITY_EOS_CUDA_VERSION: "11.6.0"
+  SINGULARITY_EOS_CUDA_VERSION: "11.4.2"
```

Comment:
I need to change this back, but other than that, can you live with these changes @ktsai7 @Yurlungur ?

Reply:
I would probably say to use a global `SINGULARITY_EOS_BUILD_TYPE` variable and use anchors to change the value in the appropriate jobs